### PR TITLE
preparing v0.5.2

### PR DESCRIPTION
**Pull request by raphaeldussin**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,15 @@
 What's new
 ==========
 
+0.5.2 (01-20-2011)
```
Comment (aulemahal) [inline CHANGES.rst:4]:
```suggestion
0.5.2 (01-20-2021)
```

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+------------------
+
+Bug fixes
+~~~~~~~~~
+
+* Restore original behavior for lon/lat discovery, uses cf-xarray if lon/lat not found in dataset (:pull:`64`)
+* Solve issue of dimension order in dataset (#53) with (:pull:`66`)
+
 0.5.1 (01-11-2011)
```
Comment (aulemahal) [inline CHANGES.rst:13]:
```suggestion
0.5.1 (01-11-2021)
```

**Review by aulemahal — COMMENTED** [review]
We shouldn't live in the past. ;)

File: CHANGES.rst (hunk 1 above)
Comment (raphaeldussin) [reply CHANGES.rst:4]: you are absolutely right! :)

**Comment by raphaeldussin** [comment]
> We shouldn't live in the past. ;)

and copy/paste :D

**Comment by raphaeldussin** [comment]
@huard I'd love to get this released so we don't distribute a buggy version as the latest version

**Review by huard — APPROVED** [review]

**Pull request merged**